docs/sources/http_api/alerting.md

```diff
@@ -7,6 +7,8 @@ aliases = ["/docs/grafana/latest/http_api/alerting/"]
 
 # Alerting API
 
+>**Note:** This information is for the [legacy dashboard alerts]({{< relref "../alerting/old-alerting/_index.md" >}}). A preview of Grafana 8 Alerts API specification can be found [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/pkg/services/ngalert/api/tooling/post.json).
```

Comment:
This link is for the upstream version of the API. For the v8.0.x (latest) docs version should be:
`https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/v8.0.x/pkg/services/ngalert/api/tooling/post.json` instead.

I wonder if there is a way to detect the target version of the docs (eg from the URI) and replace it appropriately.
Alternatively, we need to maintain this separately for each version which is harder and error prune.

Reply:
Since the new 8.0 API is an opt in feature, detecting the version is not needed at this time.